File: wf-admin/src/entities/employee.ts
export class Employee {
    id: string;
    unionid: string;
    name: string;
    mobile: string;
    email: string;
    active: boolean;
    department: string;
    isLeaderInDepts: string;
    position: string;
    avatar: string;
    hiredDate: Date;
    jobNumber: string;
    creationTime: Date;
    roleSelected: boolean;
    constructor(data?: any) {
        if (data) {
            for (var property in data) {
                if (data.hasOwnProperty(property))
                    (<any>this)[property] = (<any>data)[property];
            }
        }
    }
    init(data?: any) {
        if (data) {
            this.id = data["id"];
            this.unionid = data["unionid"];
            this.name = data["name"];
            this.mobile = data["mobile"];
            this.email = data["email"];
            this.active = data["active"];
            this.department = data["department"];
            this.isLeaderInDepts = data["isLeaderInDepts"];
            this.position = data["position"];
            this.avatar = data["avatar"];
            this.hiredDate = data["hiredDate"];
            this.jobNumber = data["jobNumber"];
            this.creationTime = data["creationTime"];
        }
    }
    toJSON(data?: any) {
        data = typeof data === 'object' ? data : {};
        data["id"] = this.id;
        data["unionid"] = this.unionid;
        data["name"] = this.name;
        data["mobile"] = this.mobile;
        data["email"] = this.email;
        data["active"] = this.active;
        data["department"] = this.department;
        data["isLeaderInDepts"] = this.isLeaderInDepts;
        data["position"] = this.position;
        data["avatar"] = this.avatar;
        data["hiredDate"] = this.hiredDate;
        data["jobNumber"] = this.jobNumber;
        data["creationTime"] = this.creationTime;
        return data;
    }
    static fromJS(data: any): Employee {
        let result = new Employee();
        result.init(data);
        return result;
    }
    static fromJSArray(dataArray: any[]): Employee[] {
        let array = [];
        dataArray.forEach(result => {
            let item = new Employee();
            item.init(result);
            array.push(item);
        });
        return array;
    }
    clone() {
        const json = this.toJSON();
        let result = new Employee();
        result.init(json);
        return result;
    }
}

File: WF.RetailCloud/aspnet-core/src/WF.RetailCloud.Core/DingTalk/Employees/Employee.cs
using Abp.Domain.Entities;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Text;

namespace WF.RetailCloud.DingTalk.Employees
{
    [Table("Employees")]
    public class Employee : Entity<string> //注意修改主键Id数据类型
    {
        /// <summary>
        /// 员工在当前开发者企业账号范围内的唯一标识
        /// </summary>
        [StringLength(200)]
        public virtual string Unionid { get; set; }
        /// <summary>
        /// 姓名
        /// </summary>
        [StringLength(50)]
        public virtual string Name { get; set; }
        /// <summary>
        /// 手机号
        /// </summary>
        [StringLength(11)]
        public virtual string Mobile { get; set; }
        /// <summary>
        /// 电子邮箱
        /// </summary>
        [StringLength(100)]
        public virtual string Email { get; set; }
        /// <summary>
        /// 是否激活
        /// </summary>
        public virtual bool? Active { get; set; }
        /// <summary>
        /// 成员所属部门id列表
        /// </summary>
        [StringLength(300)]
        public virtual string Department { get; set; }
        /// <summary>
        /// 在对应的部门中是否为主管，格式{key：部门Id，val: true/false 是/否}
        /// </summary>
        [StringLength(300)]
        public virtual string IsLeaderInDepts { get; set; }
        /// <summary>
        /// 职务信息
        /// </summary>
        [StringLength(100)]
        public virtual string Position { get; set; }
        /// <summary>
        /// 头像url
        /// </summary>
        [StringLength(200)]
        public virtual string Avatar { get; set; }
        /// <summary>
        /// 入职时间
        /// </summary>
        public virtual DateTime? HiredDate { get; set; }
        /// <summary>
        /// 员工工号
        /// </summary>
        [StringLength(100)]
        public virtual string JobNumber { get; set; }
        /// <summary>
        /// 创建时间
        /// </summary>
        [Required]
        public virtual DateTime CreationTime { get; set; }
    }

}

